tests/unit/gapic/recommendationengine_v1beta1/test_catalog_service.py

```diff
@@ -2589,7 +2592,7 @@ def test_parse_catalog_path():
 
 
 def test_common_billing_account_path():
-    billing_account = "squid"
+    billing_account = "whelk"
```

Comment:
Very odd that the generator is changing the dummy values here:  can somebody point to a source of truth for those changes?